utils.js

```diff
@@ -0,0 +1,22 @@
+const axios = require('axios');
```

Comment:
Since we use `paragon CLI` in production `axios` it is required to have in `dependencies` instead of `devDependencies`.

Reply:
Done